cmd/otel-allocator/watcher/promOperator.go

```diff
@@ -57,7 +57,16 @@ func NewPrometheusCRWatcher(cfg allocatorconfig.Config, cliConfig allocatorconfi
 		monitoringv1.PodMonitorName:     podMonitorInformers,
 	}
 
-	generator, err := prometheus.NewConfigGenerator(log.NewNopLogger(), &monitoringv1.Prometheus{}, true) // TODO replace Nop?
+	// TODO: We should make these durations configurable
```

Comment:
Do we have an open issue for this? We have the ability to configure the collector's interval for scraping the TA, we should probs have a field for this.